src/callbacks_step/analysis_surface_integral_2d.jl

```diff
@@ -76,25 +92,25 @@ which stores the boundary information and semidiscretization.
 - `aoa::Real`: Angle of attack in radians (for airfoils etc.)
 - `rhoinf::Real`: Free-stream density
 - `uinf::Real`: Free-stream velocity
-- `l_inf::Real`: Reference length of geometry (e.g. airfoil chord length)
+- `linf::Real`: Reference length of geometry (e.g. airfoil chord length)
 """
-function LiftCoefficientPressure(aoa, rhoinf, uinf, l_inf)
+function LiftCoefficientPressure(aoa, rhoinf, uinf, linf)
     # psi_lift is the normal unit vector to the freestream direction.
     # Note: The choice of the normal vector psi_lift = (-sin(aoa), cos(aoa))
     # leads to positive lift coefficients for positive angles of attack for airfoils.
     # One could also use psi_lift = (sin(aoa), -cos(aoa)) which results in the same
     # value, but with the opposite sign.
     psi_lift = (-sin(aoa), cos(aoa))
-    return LiftCoefficientPressure(ForceState(psi_lift, rhoinf, uinf, l_inf))
+    return LiftCoefficientPressure(ForceState(psi_lift, rhoinf, uinf, linf))
 end
 
 """
-    DragCoefficientPressure(aoa, rhoinf, uinf, l_inf)
+    DragCoefficientPressure(aoa, rhoinf, uinf, linf)
 
 Compute the drag coefficient
 ```math
 C_{D,p} \\coloneqq \\frac{\\oint_{\\partial \\Omega} p \\boldsymbol n \\cdot \\psi_D \\, \\mathrm{d} S}
-                        {0.5 \\cdot \\rho_{\\infty} \\cdot U_{\\infty}^2 \\cdot L_{\\infty}}
```

Comment:
I removed the dots to avoid confusion as we use the proper meaning of the dot in this formula as well.